index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Registration Form</title>
+    <link rel="stylesheet" href="css/style.css">
+    
+  </head>
+
+  <body>
+    <header>
+      <h1 id="title">Microverse Registration Form</h1>
+      <p id="description">Please register your information</p>
+    </header>
+      <form id="suervey-form">          
+   <div>
```

Comment:
If your form has multiple parts, it is better to use a fieldset tag instead of div tags to improve readability.

Reply:
Hi @ommarz 👋,

Kindly follow the suggestion, asked by the previous reviewer.
- [ ] Please use kind words, be clear, and always give a clear explanation of why the requested change is necessary.

For example, you could say: ⬇️

 Good job so far, 👏 , however, Please ensure that you use fieldset instead of <div> While <div> elements are versatile and can be used for general grouping or layout purposes, using a element when working with forms provides additional benefits in terms of accessibility, visual grouping, semantics, and form submission. It is recommended to use <fieldset> and <legend> elements when you have a logical grouping of form controls.